Tests/fixtures/adjustments/show-200.xml

```diff
@@ -12,6 +12,12 @@ Content-Type: application/xml; charset=utf-8
   <description>$12 Annual Subscription</description>
   <accounting_code></accounting_code>
   <product_code nil="nil"></product_code>
+  <custom_fields>
```

Comment:
In some cases (it happens for ruby) when the element does not have the type=array then it is not parsed as an array, but as a single element. So I suggest to add the type to comply with the actual response:
```suggestion
  <custom_fields type="array">
```